test/BasicAccessTest.php

```diff
@@ -46,7 +46,7 @@ protected function setUp()
         };
     }
 
-    public function testConstructor()
+    public function testConstructor(): void
```

Comment:
We don't generally add return type hints within test classes, unless the method is a data provider or a helper method. For test cases, it's not necessary. Please remove it here and elsewhere where you have added them.